notebooks/quickstart/data-scientist/02-explore-dataset.ipynb

```diff
@@ -0,0 +1,481 @@
+{
```

Comment:
<p>Break this into different cells.</p><p> </p><pre class="ql-syntax" spellcheck="false"># Get a pointer to the age data
ages = family_age_dataset["age"]
</pre><p> </p><p>Instead of doing a publish on the data itself, perform a query first e.g. mean.</p><p> </p><p>So, let's us now perform some analysis on the age data. For this tutorial we will calculate the mean age.</p><pre class="ql-syntax" spellcheck="false">mean_age = ages.mean()

</pre><p> </p><p>Publish and .get should be a different Step.</p><p> </p><p>Step 10:</p><p>ages_published_using_privacy_budget=ages.publish(sigma=20)</p><p> </p><p>Step 11:</p><p>ages_published_using_privacy_budget.get()</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/7045/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>